codecarbon/emissions_tracker.py

```diff
@@ -208,8 +211,7 @@ class OfflineEmissionsTracker(BaseEmissionsTracker):
     Offline implementation of the `EmissionsTracker`
     In addition to the standard arguments, the following are required.
     """
-
-    @suppress(Exception)
+    
```

Comment:
We want to suppress any exceptions or errors from Codecarbon so that it does not fail user code. @acatovic

Reply:
Thanks for the comments @kngoyal. In that case my proposal for raising an exception will not work. What do you think of these two alternatives:

- Print a Python warning "CODECARBON : Hardware list empty - no supported hardware found - emitting emissions.csv"

- sys.exit("CODECARBON : Hardware list empty - no supported hardware found - emitting emissions.csv")

The first one still lets users know that no supported hardware is found, but does not impact code flow. The 2nd one is a hard exit.

Also I suggest making the warning/error string more generic, since I'm assuming CodeCarbon will not work on ARM, Power, M1 (new Apple machines) etc, correct?